### DISCUSSION TITLE
Installing the Berea Church Organ

### DISCUSSION THREAD

**Discussion by baftimii** [GrandOrgue/grandorgue#1470] [Help]
I have downloaded the organ and patches and unzipped them but Grandorgue doesn't find the organ.  I must be missing a step or two??

**Comment by oleg68** [comment]
1. Unpack the archive in some directory.
2. File->Open and select the `.organ` file in the directory.

**Comment by baftimii** [comment]
Thank you.  I now have the original organ working.  So far, the patches don't want to unzip into the same folder.  I will work on it more tomorrow.


-----Original Message-----

**Comment by baftimii** [comment]
Now have the extended version up and running.  I can't figure out how to set the pistons??  Thanks.


-----Original Message-----

**Comment by baftimii** [comment]
Thank you for your help.  I have the organ playing and am enjoying it.  Still a bit of latency I haven't gotten rid of.  I have an acquaintance who is a sound engineer that is looking for someone to program the organ samples he has for use with Grandorgue.  Know anyone?


-----Original Message-----
